pgmpy/factors/continuous/ContinuousFactor.py

```diff
@@ -0,0 +1,109 @@
+import six
+
+
+class ContinuousFactor(object):
```

Comment:
@ankurankan I understand the implementation for marginalize. How should reduce, product, add etc. be defined?